libs/features/mas/web-components/src/variants/ccd-suggested.js

```diff
@@ -25,7 +25,10 @@ export class CCDSuggested extends VariantLayout {
       return html`
           <div style="${this.stripStyle}" class="body">
               <div class="header">
-                <slot name="icons"></slot>
+                <div class="top-section">
+                  <slot name="icons"></slot> 
```

Comment:
should we rename the slot to mnemonics?

Reply:
i'd say yes, but then we need to touch all variants? it's 'icons' across the whole thing

Reply:
maybe it's my laziness, but imo we for now could just rename the authoring UI and keep them icons in code, since it's clear enough what it is. but up to you

Reply:
we have a mechanism already to map properties to slot names: 
https://github.com/adobecom/milo/blob/stage/libs/features/mas/web-components/src/variants/ccd-suggested.js#L10

so we can afford mnemonic slot and map it correctly.
later  it would be good, when someone inspects the DOM and sees icon instead of mnemonic.